types/firestore.d.ts

```diff
@@ -778,9 +778,11 @@ declare namespace FirebaseFirestore {
 
   /**
    * Filter conditions in a `Query.where()` clause are specified using the
-   * strings '<', '<=', '==', '>=', '>', and 'array-contains'.
+   * strings '<', '<=', '==', '>=', '>', 'array-contains', 'in', and 
+   * 'array-contains-any'.
    */
-  export type WhereFilterOp = '<' | '<=' | '==' | '>=' | '>' | 'array-contains';
+  export type WhereFilterOp = '<' | '<=' | '==' | '>=' | '>' | 'array-contains' | 
```

Comment:
Do you also mind expanding the compile-time test here: https://github.com/googleapis/nodejs-firestore/blob/d15f29d71b27e867b192bcffa2ee7b75c5429d9f/dev/test/typescript.ts#L240

Reply:
I did that above already. Do you mean to add more items into the array, or validate in other ways?

Reply:
That's what I wanted. Never mind.